src/context.rs

```diff
@@ -3529,7 +3518,23 @@ impl<'a> ContextWriter<'a> {
     };
 
     assert!(mode >= PredictionMode::NEAREST_NEARESTMV);
-    let val = mode as u32 - PredictionMode::NEAREST_NEARESTMV as u32;
+    let val = match mode {
```

Comment:
what about making a small method `to_u8` or such?

Reply:
can't derive #[Debug] for arrays larger than 32

Reply:
could but this is the only case where this numeric conversion is used

Reply:
Ok.